source/postprocess/visualization.cc

```diff
@@ -209,6 +211,8 @@ namespace aspect
       // initialize this to a nonsensical value; set it to the actual time
       // the first time around we get to check it
       last_output_time (std::numeric_limits<double>::quiet_NaN()),
+      maximum_timesteps_between_outputs (numbers::invalid_unsigned_int),
```

Comment:
I think you want this to be `std::numeric_limits<int>::max()` to match the parameter's default.